corpustools/adder.py

```diff
@@ -66,9 +67,22 @@ def url_to_filename(response):
         _, params = cgi.parse_header(response.headers["Content-Disposition"])
         return params["filename"]
     except KeyError:
-        return add_url_extension(
-            os.path.basename(response.url), response.headers["content-type"]
-        )
+        try:
```

Comment:
Det å lage navnet til dokumentet bør flyttes til crawler.py, under funksjonen save_pages. Der kan du f.eks. erstatte innholdet linjene i 54-71 med denne koden, og bruke [copy_url_to_corpus](https://giellalt.github.io/CorpusTools/reference/adder/#corpustools.adder.AddToCorpus.copy_url_to_corpus) og bruke argumentet wanted_name for å gi navnet som blir laget av denne koden